content/events-api/receiving-events.md

```diff
@@ -0,0 +1,30 @@
+# Receiving Events
+
+Your Event API Request URL will receive events matching your subscriptions. An event could be delivered with a dedicated message or in a message that gathers several events. 
+
+Be careful as some events can be sent in the wrong order, more than one time or get lost altogether.
+
+This is why the `event_id` and `event_date` are important:
+
+- The `event_id` allows you to track the events you have already processed. You will not receive events older than **1 hour**, so no need to keep track of them past that.
+- The `event_date` allows you to not processing an event if you receive an older event after a newer one.
+
+## Events API Request URLs
+
+Request URL receives an HTTP POST containing data in response to activity.
+
+In the Events API, your Events API Request URL is the target location where all the events your connection is subscribed to will be delivered, regardless of the event type.
+
+Since your connected application will have only one Request URL, maybe you'll need to do an additional dispatch or routing server-side after receiving event data.
+
+Your Request URL will receive JSON-based payloads containing wrapped event types.
+
+## Performance considerations
```

Comment:
I don't know if it's a good thing to write this part here. Maybe the dedicated "best practices / advices" section would be better?

My concern is that it's far more complex than what is written here and it's not only the volume of events but also the volume of API events we need to send. It's not only a "receiving events" topic. It's also about the batch size, the nightly sync, the queuing system, number of activated events subscription, and it brings some topics like limitations, retention duration, optimization
Also, there are a lot of customizable things that would help in solving problems